RecoLocalCalo/CaloTowersCreator/interface/EScales.h

```diff
@@ -2,20 +2,20 @@
 #define EScales_h
 
 struct EScales{
-  double HBPiOvere;
-  double HESPiOvere;
-  double HEDPiOvere;
-  double HOPiOvere;
-  double HF1PiOvere;
-  double HF2PiOvere;
-  double EBScale;
-  double EEScale;
-  double HBScale;
-  double HESScale;
-  double HEDScale;
-  double HOScale;
-  double HF1Scale;
-  double HF2Scale;
+  double HBPiOvere=50.0;
```

Comment:
please set these to 0, as they were implicitly intended in original development (just check the implementation e.g. in https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_0_pre4/RecoLocalCalo/CaloTowersCreator/src/CaloTowersCreationAlgo.cc#L1241).
This will agree with the original logic of the code.
Having non-zero values can break the behavior eventually (the assignment logic it pretty brittle in the CaloTowers producers and algo).